client/my-sites/plugins/plugin.jsx

```diff
@@ -96,9 +97,13 @@ const SinglePlugin = createReactClass( {
 			sitePlugin
 		);
 
+		const notInstalledSites = props.sitesWithoutPlugin.map( ( siteId ) =>
```

Comment:
What about renaming this to something more descriptive like `sitesWithoutPlugin` or `(pluginN|n)otInstalledOnSites`? Or does it make sense to move that `.map` call to selector space as well?

Reply:
If you ask me, we should completely get rid of this component state here, and rely solely on props. I'm actually inclined to refactor a bunch more in these components, but I'm trying to keep the changes targeted and self-contained. I think I'll keep it as-is for now, but we may want to refactor it in the future and remove some of the unnecessary component state that needlessly duplicates props.